tutorcodejail/tests/test_plugin.py

```diff
@@ -0,0 +1,22 @@
+"""Codejail plugin test"""
+import unittest
+
+from tutorcodejail.plugin import patches
+
+
+class CodejailPluginTest(unittest.TestCase):
+    """
+    Test case for tutor codejail plugin.
+    """
+
+    def test_patches(self):
+        """
+        Test total patches for codejail plugin.
+
+        Expected results:
+            - Total patches expected from patches dir
```

Comment:
instead of checking the number, you could which one returns

Reply:
Done